jenkins-design-language/src/js/components/material-ui/svg-icons/action/accessibility.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import pure from 'recompose/pure';
```

Comment:
I left all the icons as they are declared in "https://github.com/callemall/material-ui" because it would make it really easy to upgrade all the icons in the future, I don't really understand why they decided to use the recompose/pure component but if we want to remove it, we have to remove it from every svg and every time we want to upgrade the icons, remember to remove it again from the upgraded icons